examples/tutorials/advanced/dataset_on_surface.py

```diff
@@ -0,0 +1,167 @@
+"""
+Draping a dataset on top of a topographic surface
+==================================================
+
+It can be visually appealing to "drape" a dataset over a topographic surface.
+This can be accomplished using the ``drapegrid`` parameter of
+:meth:`pygmt.Figure.grdview`.
+
+This tutorial consists of two examples:
+
+1. Draping a grid
+
+2. Draping an image
+"""
+
+# %%
+
+# Load the required packages
+import pandas as pd
+import pygmt
+import rasterio
+import xarray as xr
+
+# %%
+# 1. Drapping a grid
+# ------------------
+#
+# In the first example, the seafloor crustal age is plotted with color-coding
+# on top of the topographic map of an area of the Mid-Atlantic Ridge.
+
+# Define study area
+region_2d = [-50, 0, 36, 70]  # [lon_min, lon_max, lat_min, lat_max]
```

Comment:
```suggestion
# Define the study area in the form of [lon_min, lon_max, lat_min, lat_max]
region_2d = [-50, 0, 36, 70]
```

Reply:
Related to  commit https://github.com/GenericMappingTools/pygmt/pull/3316/commits/194b7bccdd72e53853135155b063a36feca90ef7.